front/src/utils/date.ts

```diff
@@ -53,6 +53,8 @@ export const formatLocalDateTime = (date: Date) =>
  */
 export const formatLocalDate = (date: Date) => dayjs(date).local().format('YYYY-MM-DD');
 
+export const formatLocalTime = (date: Date) => dayjs(date).local().format('HH:mm:ss');
```

Comment:
Could we add a small doc comment, indicating the purpose of this function, to avoid having it extended/used for other things?

```ts
/**
 * Format a local date suitable for an <input type="time">.
 */
```

Reply:
done